### PR TITLE
feat: renaming tokens to API tokens

### PR DESCRIPTION
**Pull request by russorat**
Closes https://github.com/influxdata/ui/issues/2199

Along with https://github.com/influxdata/docs-v2/pull/2929 this changes anywhere we display the text tokens to be API token

It also fixes an issue with the heights of the token/bucket selector on the client libs page.

### CODE REVIEW

File: src/writeData/components/WriteDataHelperTokens.tsx [hunk 1]
```diff
@@ -78,7 +79,7 @@ const WriteDataHelperTokens: FC = () => {
             wrapText={true}
             gradient={Gradients.GundamPilot}
           >
-            {t.description}
+            {t.description || DEFAULT_TOKEN_DESCRIPTION}
```
Comment (russorat) [inline src/writeData/components/WriteDataHelperTokens.tsx:82]: fix for https://github.com/influxdata/ui/issues/2199

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 61 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2200)

File: src/authorizations/components/GenerateTokenDropdown.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ const GenerateTokenDropdown: FC<GenerateTokenProps> = ({history}) => {
   return (
     <Dropdown
       testID="dropdown--gen-token"
-      style={{width: '160px'}}
+      style={{width: '180px'}}
```
Comment (russorat) [inline src/authorizations/components/GenerateTokenDropdown.tsx:42]: button needs to be larger to account for the new text

File: src/writeData/components/WriteDataHelperBuckets.tsx [hunk 3]
```diff
@@ -59,8 +59,8 @@ const WriteDataHelperBuckets: FC = () => {
     body = (
       <List
         backgroundColor={InfluxColors.Obsidian}
-        style={{height: '118px'}}
-        maxHeight="118px"
+        style={{height: '200px'}}
+        maxHeight="200px"
```
Comment (russorat) [inline src/writeData/components/WriteDataHelperBuckets.tsx:63]: fixes this: 
image

**Review by genehynson — APPROVED** [review]

**Review by Palakp41 — APPROVED** [review]
Looks good!

**Pull request merged**